contracts/eosio.system/src/rex.cpp

```diff
@@ -616,6 +616,64 @@ namespace eosiosystem {
 
    }
 
+   /**
+    * @brief Adds returns from the REX return pool to the REX pool
+    */
+   void system_contract::update_rex_pool()
+   {
```

Comment:
Create a const variable for the first pool element, `const auto ret_pool_elem = _rexretpool.begin()`.  These are not be collapsed and it is calling the begin function multiple times.

Reply:
Done.